senders/pushover/pushover.go

```diff
@@ -71,21 +75,34 @@ func (sender *Sender) makePushoverMessage(events moira.NotificationEvents, conta
 	return pushoverMessage
 }
 
-func (sender *Sender) buildMessage(events moira.NotificationEvents, throttled bool) string {
+func (sender *Sender) buildMessage(events moira.NotificationEvents, throttled bool, trigger moira.TriggerData) string {
 	var message bytes.Buffer
-	for i, event := range events {
-		if i > printEventsCount-1 {
-			break
-		}
-		message.WriteString(fmt.Sprintf("%s: %s = %s (%s to %s)", event.FormatTimestamp(sender.location), event.Metric, event.GetMetricValue(), event.OldState, event.State))
+	desc := trigger.Desc
+	if len(desc) > descLimit {
+		// trim description to fit 512 chars while leaving space for html tags
+		desc = desc[:descLimit-100] + "...\n"
+	}
+	htmlDesc := blackfriday.Run([]byte(desc))
+	htmlDescWithbr := strings.Replace(string(htmlDesc), "\n", "\n<br/>", -1)
```

Comment:
Why do we need both `\n` and `<br/>` in resulting text? Maybe just `<br/>` is enough?

Reply:
As of Go 1.11, the compiler optimizes this pattern to behave the same as utf8.RuneCountInString(string)
https://github.com/golang/go/issues/24923